README2.md

```diff
@@ -1,380 +0,0 @@
-# xero-node
```

Comment:
Is this a deletion of the whole README2 file from git? Looks like just the content is empty now not the file?